pkgs/development/python-modules/generic/default.nix

```diff
@@ -112,7 +112,7 @@ python.stdenv.mkDerivation (builtins.removeAttrs attrs ["disabled" "doCheck"] //
   postFixup = attrs.postFixup or ''
     wrapPythonPrograms
 
-    # check if we have two packagegs with the same name in closure and fail
+    # check if we have two packages with the same name in closure and fail
     # this shouldn't happen, something went wrong with dependencies specs
```

Comment:
if we move comments out of the arguments, then future changes should not cause any rebuilds, right?